Makefile

```diff
@@ -16,7 +16,7 @@ vendor:
 	@go mod vendor
 
 .PHONY: cross
-cross: amd64 386 arm arm64 ## build cross platform binaries
+cross: amd64 386 arm arm64 s390x ## build cross platform binaries
```

Comment:
Could you please update the README.md as well?

Reply:
@sthaha I don't see how I can update README.md here, as Source install works as is, and so far there is no s390x build to add to tarball, deb or rpm sections.